tg/views/__init__.py

```diff
@@ -155,7 +156,8 @@ def _chat_attributes(self, is_selected: bool = False) -> Tuple[int]:
 
     def draw(self, current: int, chats: List[Dict[str, Any]]) -> None:
         self.win.erase()
-        self.win.vline(0, self.w - 1, curses.ACS_VLINE, self.h)
+        line = curses.ACS_VLINE  # type: ignore
+        self.win.vline(0, self.w - 1, line, self.h)
```

Comment:
I don't like it, can it be disabled in mypy? It's forcing you to write some extra code which is stupid.

Reply:
mypy is not ideal, here it seems that it contains outdated type stub for `curses` and it things that `ACS_VLINE` is not existing and we will get error.
The only way how to disabled it is to `type: ignore`.
Or you could disable all type check by stop using mypy completley :man_shrugging: